Libplanet.Action/ActionEvaluator.cs

```diff
@@ -148,39 +148,18 @@ public class ActionEvaluator : IActionEvaluator
         /// <param name="previousState">The states immediately before <paramref name="actions"/>
         /// being executed.</param>
         /// <param name="actions">Actions to evaluate.</param>
+        /// <param name="stateStore">An <see cref="IStateStore"/> to use.</param>
         /// <param name="logger">An optional logger.</param>
         /// <returns>An enumeration of <see cref="ActionEvaluation"/>s for each
         /// <see cref="IAction"/> in <paramref name="actions"/>.
         /// </returns>
-        /// <remarks>
-        /// <para>Each <see cref="IActionContext.Random"/> object has an unconsumed state.</para>
-        /// <para>
-        /// The returned enumeration has the following properties:
-        /// <list type="bullet">
-        /// <item><description>
-        ///     The first <see cref="ActionEvaluation"/> in the enumerated result,
-        ///     if any, has <see cref="ActionEvaluation.OutputState"/> with
-        ///     <see cref="IAccount.Delta"/> that is a
-        ///     "superset" of <paramref name="previousState"/>'s
-        ///     <see cref="IAccount.Delta"/> (possibly except for
-        ///     <see cref="IAccountDelta.ValidatorSet"/>).
-        /// </description></item>
-        /// <item><description>
-        ///     Each <see cref="ActionEvaluation"/> in the enumerated result
-        ///     has <see cref="ActionEvaluation.OutputState"/> with
-        ///     <see cref="IAccount.Delta"/> that is a "superset"
-        ///     of the previous one, if any (possibly except for
-        ///     <see cref="IAccountDelta.ValidatorSet"/>).
-        /// </description></item>
-        /// </list>
-        /// </para>
-        /// </remarks>
         [Pure]
         internal static IEnumerable<ActionEvaluation> EvaluateActions(
```

Comment:
As we said, is `static` necessary...?

Reply:
It's probably better to make it non-`static`. Changing requires changes to test code. Being lazy is all. 🙄
I'll deal with it in a separate PR if I get the time.

Reply:
> Being lazy is all. 🙄

Agree.